### ISSUE TITLE
HDFilmCehennemi: domain changed 

### ISSUE THREAD

**Issue by ghost** [Kohi-den/extensions-source#282]
### Source information

HDFilmCehennemi 14.16

### Source language

Turkish

### Source new URL

https://www.hdfilmcehennemi.sh/

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I will fill out all of the requested information in this form.

**Comment by kyoyacchi** [comment]
No. It is not about their domain change. They changed the website ui completely. See #31
